packages/blade/src/components/Tag/Tag.tsx

```diff
@@ -1,72 +1,30 @@
 import React from 'react';
 import styled from 'styled-components';
+import type { TagProps } from './types';
 import { Box } from '~components/Box';
-import type { StyledPropsBlade } from '~components/Box/styledProps';
 import { getStyledProps } from '~components/Box/styledProps';
 import { IconButton } from '~components/Button/IconButton';
-import type { IconComponent } from '~components/Icons';
 import { CloseIcon } from '~components/Icons';
 import { Text } from '~components/Typography';
 import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
-import type { StringChildrenType, TestID } from '~utils/types';
 import { isReactNative, makeSize } from '~utils';
 import { size as globalSizeTokens } from '~tokens/global';
 import BaseBox from '~components/Box/BaseBox';
 import type { PaddingProps } from '~components/Box/BaseBox/types/spacingTypes';
 import { useIsMobile } from '~utils/useIsMobile';
 
-type TagProps = {
-  /**
-   * Decides the size of Tag
-   *
-   * @default medium
-   */
-  size?: 'medium' | 'large';
-
-  /**
-   * Leading icon for your Tag
-   */
-  icon?: IconComponent;
-
-  /**
-   * Callback when close icon on Tag is clicked
-   */
-  onDismiss: () => void;
-
-  /**
-   * Text that renders inside Tag
-   */
-  children: StringChildrenType;
-
-  /**
-   * Disable tag
-   */
-  isDisabled?: boolean;
-
-  /**
-   * Private property for Blade.
-   *
-   * Should not be used by consumers.
-   *
-   * Used for adding virtual focus on tag.
-   *
-   * @private
-   *
-   */
-  _isTagFocussed?: boolean;
-} & StyledPropsBlade &
-  TestID;
-
-const FocussableTag = styled(BaseBox)<{ isTagFocussed: TagProps['_isTagFocussed'] }>((props) => {
-  if (props.isTagFocussed) {
-    return {
-      outline: `1px solid ${props.theme.colors.surface.background.level1.lowContrast}`,
-      boxShadow: `0px 0px 0px 4px ${props.theme.colors.brand.primary[400]}`,
-    };
-  }
+const FocussableTag = styled(BaseBox)<{ _isVirtuallyFocussed: TagProps['_isVirtuallyFocussed'] }>(
+  (props) => {
+    if (props._isVirtuallyFocussed && !isReactNative()) {
+      return {
+        outline: `1px solid ${props.theme.colors.surface.background.level1.lowContrast}`,
```

Comment:
We  could use 1px border radius token?

Reply:
This isn't border. This is for focus ring. We have another task in issues to standardise focus rings so can fix it then.

For now I changed it to use size token

Reply:
https://github.com/razorpay/blade/pull/1459#discussion_r1321012238 this will also be fixed with focus ring standardisation